diagnostic.md

```diff
@@ -23,7 +29,8 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+ const parent = $(.listerItem).parent()
+ parent.siblings('form')
```

Comment:
I think you have to go up another level here: `$('li').parent().parent().find('form')`